app/models/user.rb

```diff
@@ -63,4 +68,10 @@ def create_activation_digest
     self.activation_token = User.new_token
     self.activation_digest = User.digest(activation_token)
   end
+
+  def picture_size
+    if picture.size > 5.megabytes
+      errors.add(:picture, "should be less than 5MB")
```

Comment:
()

Reply:
errors.add(:picture, "should be less than 5MB") if ......

Reply:
chưa fix comment à